### ISSUE TITLE
What is the accuracy reported on? 

### ISSUE THREAD

**Issue by duygusar** [jeffreyhuang1/two-stream-action-recognition#53]
In the repository I think the test set refers to evaluation set, is this correct? The evaluation set is not partitioned from the training set. I am skimming through the code and I think test actually refers to evaluations set and if you needed an actual test you need to replace the test split with a new one (with unseen examples), I just found it peculiar and wanted to make sure if I am correct about this. So, I am confused about the reported accuracy because a real test split is not provided. Is the accuracy on README the validation accuracy?
